src/App.css

```diff
@@ -1,38 +1,266 @@
-.App {
-  text-align: center;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;1,100;1,200;1,300;1,400&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Montserrat;
+  background-color: #f5f6fa;
+}
+
+header {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  padding: 1rem 5rem;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  align-items: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+h4 {
+  color: #121212;
+  opacity: 0.5;
+  margin-bottom: 0.5rem;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+h2 {
+  margin: 0.5rem 0;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+h5 {
+  margin-top: 0.5rem;
+  color: #4386bf;
+  font-weight: 400;
+}
+
+.nav-items {
   display: flex;
-  flex-direction: column;
+  justify-content: space-between;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-content: center;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+}
+
+.logo {
+  color: #0290ff;
+  font-size: 2rem;
+  margin: 0;
+}
+
+li {
+  margin: 0;
+  padding: 0 1rem;
+  font-size: 0.85rem;
+  font-weight: 600;
+}
+
+.books {
+  color: #121212;
+}
+
+.categories {
+  color: #121212;
+  border-radius: 4px;
+  background-color: #fafafa;
+  opacity: 0.5;
+}
+
+.fa-user {
+  padding: 1rem;
+  font-size: 1rem;
+  color: #0290ff;
+  border: 1px #e8e8e8 solid;
+  border-radius: 50%;
+  text-align: center;
+}
+
+.container {
+  padding: 3rem 5rem;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+}
+
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
```

Comment:
no repeating of the same rule
Please correct those issues and send another review request, good luck, and well done so far!

Reply:
**Thank you for your review** 😄

- The element is stretching to take the full width of the screen when you zoom and the screen size gets changed! In the normal 100% screen size, I think my design is ok!
- There's not any repetition! they're `align-items` and `align-content`!!